examples/arrays.md

```diff
@@ -0,0 +1,81 @@
+# Arrays
+
+An array is a collection of items stored at contiguous memory locations. It's an aggregate data structure that is designed to store a group of objects of the same types. It's the most efficient data structure for storing and accessing a sequence of objects.
+
+## How to declare an array
+
+```v
+mut ages := [18, 25, 37]
+
+println(ages)
+>>> [18, 25, 37]
+
+println(ages[1])
+>>> 25
+
+mut names := ['Vitor', 'Don', 'Swastik']
+
+println(names)
+>> ['Vitor', 'Don', 'Swastik']
+
+println(names[0])
+>> Vitor
+```
+
+Note: All elements must have the same type. `['Vitor', 'Don', 'Swastik', 0]` will not compile.
+
+```bash
+>>> mut names := ['Vitor', 'Don', 'Swastik', 0]  
```

Comment:
would prefer to use our github names not real names 😛

Reply:
ya.. I agree...   😁